### PR TITLE
French big review

### PR DESCRIPTION
**Pull request by ldmpub**
fix + improvements + (some) missing translation for french language.

### CODE REVIEW

**Comment by forrestguice** [comment]
Thank you for updating this. It's just in time for the `v0.13.11` patch.

If you want to make future changes, just modify the file directly (as you've done), and submit another pull request. I'll review it and tag a patch release (typically every 2 to 4 weeks). It's typically on F-Droid a week later.

btw, if you are looking to collaborate with others on the translation, maybe creating an issue in the tracker and tagging the previous French translators might be a way to start. There hasn't been much need for a web based translation system.

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -1274,7 +1274,7 @@
     <string-array name="locale_credits" translatable="false">  <!-- one-to-one with locale_display -->
         <item />
         <item><![CDATA[<a href=https://github.com/Verdulo>Verdulo</a>]]></item>
-        <item><![CDATA[<a href=https://github.com/Jej>Jej</a>|<a href=https://github.com/Aloha>Aloha</a>]]></item>
+        <item><![CDATA[<a href=https://github.com/Jej>Jej</a>|<a href=https://github.com/Aloha>Aloha</a>|<a href=https://github.com/ldmpub>ldmpub</a>]]></item>
```
Comment (forrestguice) [inline app/src/main/res/values/strings.xml:1277]: The locale_credits are displayed in the About dialog (also copied to places like the README, etc). 

I added something in the appropriate location - put whatever you want here.

File: app/src/main/res/values-fr/strings.xml [hunk 2]
```diff
@@ -225,9 +225,9 @@
     <string name="configLabel_general_calculator_sun">Source de données pour le soleil</string>         <!-- preference label -->
     <string name="configLabel_general_calculator_moon">Source de données pour la lune</string>         <!-- preference label -->
 
-    <string name="configLabel_general_timeFormatMode">Forfait d'eheure :</string>         <!-- spinner label -->  <!-- TODO -->
+    <string name="configLabel_general_timeFormatMode">Forfait d\'eheure :</string>         <!-- spinner label -->  <!-- TODO -->
```
Comment (forrestguice) [inline app/src/main/res/values-fr/strings.xml:228]: `'` is a special character. It needs to be escaped with a `\`

Comment (ldmpub) [reply app/src/main/res/values-fr/strings.xml:228]: Thanks and sorry but there are a big typo error we need to correct, translation should be: "Format des heures"

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (ldmpub) [reply app/src/main/res/values/strings.xml:1277]: Ok I'm not familiar with claiming any credit for my translations .. do it your own way 😎.

Comment (forrestguice) [reply app/src/main/res/values/strings.xml:1277]: I like to keep track of attribution like this, but its totally optional - link to your github, no link, full name, pseudonym, or no entry at all - whatever - please don't hesitate to change this value.

File: app/src/main/res/values-fr/strings.xml (hunk 2 above)
Comment (forrestguice) [reply app/src/main/res/values-fr/strings.xml]: fixed. I picked that line at random. Lucky review I guess.

**Comment by ldmpub** [comment]
Thanks.

**Pull request merged**